src/ring_theory/power_series/basic.lean

```diff
@@ -1130,6 +1146,32 @@ lemma mul_inv_of_unit (φ : power_series R) (u : units R) (h : constant_coeff R
   φ * inv_of_unit φ u = 1 :=
 mv_power_series.mul_inv_of_unit φ u $ h
 
+open finset nat
+
+variables {A : Type*} [comm_ring A]
+
+lemma eval_mul_hom_zero (f : power_series A) : eval_mul_hom 0 f = C A (constant_coeff A f) :=
+begin
+  rw [eval_mul_hom, ring_hom.coe_mk], ext, rw [power_series.coeff_mk _ _, coeff_C],
+  split_ifs, { rw h, simp only [one_mul, coeff_zero_eq_constant_coeff, pow_zero], },
+  { rw [zero_pow' n h, zero_mul], },
+end
+
+lemma eval_mul_hom_one (f : power_series A) : eval_mul_hom 1 f = f :=
+by { rw eval_mul_hom, ext, simp only [one_pow, coeff_mk, one_mul, ring_hom.coe_mk], }
```

Comment:
This can go back up to line 1028 - it doesn't need `comm_ring`, only `comm_semiring`

Reply:
Yeah, I noticed, thank you!